docs/base.md

```diff
@@ -212,8 +213,79 @@ end
 
 ---
 
+####spawnObjectData(...)
 
+Spawns an Object using a data table. Works with [getData()](object.md#getdata). It works just like spawnObject, but instead of a `type`, you supply a `data` table. The other parameters will overwrite those in the Data.
 
+!!!tip
+	Spawned Objects take a moment to be physically spawned into the game. The purpose of the callback functionality is to allow you to run additional actions after the Object has been initiated fully into the instance. You can also add a delay after spawning using a [Wait](wait.md) function.
+
+!!!info "spawnObjectData(parameters)"
+	* [<span class="tag tab"></span>](types.md) **parameters**: A Table of parameters used to determine how spawnObjectData will act.
+		* [<span class="tag tab"></span>](types.md) **parameters.data**: [getData()](object.md#getdata) table.
+		* [<span class="tag vec"></span>](types.md#vector) **parameters.position**: Position to place Object.
+			* {>>Optional, defaults to data table's value.<<}
+		* [<span class="tag vec"></span>](types.md#vector) **parameters.rotation**: Rotation of the Object.
+			* {>>Optional, defaults to data table's value.<<}
+		* [<span class="tag vec"></span>](types.md#vector) **parameters.scale**: Scale of the Object.
+			* {>>Optional, defaults to data table's value.<<}
+		* [<span class="tag fun"></span>](types.md#function) **parameters.callback_function**: The function to activate after the Object has finished spawning into the scene.
+			* {>>Optional, defaults to not being used.<<}
+			* {>>A reference to the object spawned is always passed to callback_function. See the example for how to access it.<<}
+
+``` Lua
+function onObjectDrop(player_color, dropped_object)
+	futureName = "Spawned By Script!"
+	spawnParams = {
+		data              = dropped_object.getData(),
+		position          = {x=0, y=3, z=-5},
+		rotation          = {x=0, y=90, z=0},
+		scale             = {x=2, y=2, z=2},
+		sound             = false,
```

Comment:
Could you please remove `sound` and `snap_to_grid`. _Technically_ TTS will accept those options, however I'm pretty sure they don't do anything.

Reply:
I removed these for spawnObjectData but left them for spawnObjectJSON as you've requested changes to that be in a different pull request.